OpenNefia.Content/TitleScreen/MainTitleLogic.cs

```diff
@@ -115,6 +119,12 @@ private void StartGame()
             // new saves should always start from entity 0
         }
 
+        private void CreateChar()
+        {
+            var creatChar = IoCManager.Resolve<ICharaMakeLogic>();
```

Comment:
Could this go into a `[Dependency]` field?

Reply:
done in 76f6762